Cython/Utility/ObjectHandling.c

```diff
@@ -3239,17 +3252,15 @@ static CYTHON_INLINE void __Pyx_RaiseClosureNameError(const char *varname) {
     PyErr_Format(PyExc_NameError, "free variable '%s' referenced before assignment in enclosing scope", varname);
 }
 
+/////////////// RaiseClosureNameErrorNogil.proto ///////////////
+static CYTHON_INLINE void __Pyx_RaiseClosureNameErrorNogil(const char *varname);/*proto*/
 
-/////////////// RaiseUnboundMemoryviewSliceNogil.proto ///////////////
-static void __Pyx_RaiseUnboundMemoryviewSliceNogil(const char *varname);/*proto*/
-
-/////////////// RaiseUnboundMemoryviewSliceNogil ///////////////
-//@requires: RaiseUnboundLocalError
+/////////////// RaiseClosureNameErrorNogil ///////////////
+//@requires: RaiseClosureNameError
 
-// Don't inline the function, it should really never be called in production
-static void __Pyx_RaiseUnboundMemoryviewSliceNogil(const char *varname) {
+static CYTHON_INLINE void __Pyx_RaiseClosureNameErrorNogil(const char *varname) {
```

Comment:
What makes this special enough for inlining?

Reply:
Yeah was a copy-paste mistake on this one. But I agree - I'll remove it from the inner layer too.